docker/build-tools/Dockerfile

```diff
@@ -99,10 +99,10 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 	libc-dev \
 	make \
 	pkg-config \
-    wget \
-    && rm -rf /var/lib/apt/lists/*
+    wget
```

Comment:
nit - this line and others mixing tabs/spaces

Reply:
Fixed, all spaces now.